### PR TITLE
Setup database with seed data

### PR DESCRIPTION
**Pull request by MikeGattis**
Am I doing this right? @belzy  @jeremyengland

### CODE REVIEW

**Review by belzy — APPROVED** [review]
Looks great. Faker js looks useful so I will use this as well.

File: migrations/create-users.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      user_image_url: {
+        type: Sequelize.STRING
+      },
+      like: {
+        type: Sequelize.BOOLEAN
+      },
+      repost: {
+        type: Sequelize.BOOLEAN
+      }
+      // createdAt: {
```
Comment (jeremyengland) [inline migrations/create-users.js]: Maybe take out these comments for things you're not using.

File: config/config.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
```
Comment (jeremyengland) [inline config/config.json]: Looks great!

**Review by jeremyengland — APPROVED** [review]
Everything looks great Mike!

File: migrations/create-users.js (hunk 1 above)
Comment (MikeGattis) [reply migrations/create-users.js]: Fair point, and I will, thanks!

**Pull request closed without merging**